style.css

```diff
@@ -384,3 +426,80 @@ body {
   width: 327px;
   border-bottom: 1px solid #dfe1e6;
 }
+
+.contact-form-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  margin: 0 0 8px 0;
+  width: 375px;
+}
+
+.contact-form {
+  border-radius: 60px 0 0 0;
+  background-image: url(./img/contact-form-shapes-mobile.svg);
```

Comment:
- [x] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. The form background image is supposed to be to the right but your design it is in the center. Please move the image to the right and make sure it matches the design.

Your design

![Screenshot (114)](https://user-images.githubusercontent.com/83803414/154135254-ed772087-d557-40c2-adf8-ca7201bc0f47.png)

Figma design

![Screenshot (115)](https://user-images.githubusercontent.com/83803414/154135289-89532f88-8430-466f-b5e6-2737120f5380.png)